flipperdevices/flipperzero-firmware
# [FL-2494, FL-2497] PIN Lock refactoring, IR library fix

# What's new

- Lock system on start if PIN was set
- PIN locking code moved to desktop helpers
- IR: universal library file type change

# Verification 

- Check PIN locking
- Try to open tv.ir from archive

# Checklist (For Reviewer)
